model/roles_test.go

```diff
@@ -197,6 +198,88 @@ func TestRolesSort(t *testing.T) {
 	assert.Equal(roles[1].Name, "ddd")
 }
 
+func TestGetScriptSignatures(t *testing.T) {
+	assert := assert.New(t)
+
+	refRole := &Role{
+		Name: "bbb",
+		Jobs: Jobs{
+			{
+				SHA1: "Role 2 Job 1",
+				Packages: Packages{
+					{Name: "aaa", SHA1: "Role 2 Job 1 Package 1"},
+					{Name: "bbb", SHA1: "Role 2 Job 1 Package 2"},
+				},
+			},
+			{
+				SHA1: "Role 2 Job 2",
+				Packages: Packages{
+					{Name: "ccc", SHA1: "Role 2 Job 2 Package 1"},
+				},
+			},
+		},
+	}
+
+	firstHash, _ := refRole.GetScriptSignatures()
+
+	workDir, err := ioutil.TempDir("", "fissile-test-")
+	assert.NoError(err)
+	defer os.RemoveAll(workDir)
+	releasePath := filepath.Join(workDir, "role.yml")
+
+	scriptName := "script.sh"
+	scriptPath := filepath.Join(workDir, scriptName)
+	scriptHandle, err := os.Create(scriptPath)
```

Comment:
Probably want to use [ioutil.WriteFile](https://golang.org/pkg/io/ioutil/#WriteFile)